Directory.Build.props

```diff
@@ -1,11 +1,15 @@
 <Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
   <PropertyGroup>
     <EnableDefaultItems>false</EnableDefaultItems>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <IsPackable>false</IsPackable>
+    <IsShipping>false</IsShipping>
+    <SuppressLicenseValidation>true</SuppressLicenseValidation>
```

Comment:
is this needed because cecils LICENSE.txt doesn't fully match the template in https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Arcade.Sdk/tools/Licenses/MIT.txt ?

Reply:
Yes, that's the case